src/main/java/uk/gov/hmcts/reform/locationrefdata/camel/binder/ServiceToCcdCaseType.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.locationrefdata.camel.binder;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import org.apache.camel.dataformat.bindy.annotation.CsvRecord;
+import org.apache.camel.dataformat.bindy.annotation.DataField;
+import org.springframework.stereotype.Component;
+
+import javax.validation.constraints.NotEmpty;
+
+@Component
+@Setter
+@Getter
+@CsvRecord(separator = ",", crlf = "UNIX", skipFirstLine = true, skipField = true)
+@Builder
+@RequiredArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
+public class ServiceToCcdCaseType {
+
+    @DataField(pos = 1, columnName = "service_code")
+    @NotEmpty
+    String serviceCode;
+
+    @DataField(pos = 2, columnName = "ccd_service_name")
+    String ccdServiceName;
+
+    @DataField(pos = 3, columnName = "ccd_case_type")
+    @NotEmpty
+    String ccdCaseType;
```

Comment:
ccd case type is not empty and there is possibility the the file may not have any value in this column for a row

Reply:
done